docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -189,6 +189,22 @@ Folder that contains [provisioning]({{< relref "../../administration/provisionin
 
 `http`,`https`,`h2` or `socket`
 
+### min_tls_version
+
+The TLS Handshake requires a minimum TLS version. The available options are TLS1.2 and TLS1.3.
+If you do not specify a version, the system uses TLS1.2.
+
+### tls_ciphers
+
+This section refers to the values that are accepted for the TLS cipher. By default, this value is left empty. Example value: "TLS_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256"
+For a list of supported ciphers and TLS versions, refer to [Source file src/crypto/tls/cipher_suites.go](https://go.dev/src/crypto/tls/cipher_suites.go).
+
+To support backward compatibility
+If the protocol is set to https and there are no ciphers configured, the system will apply the following ciphers to support backward compatibility for TLS 1.2: `TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,TLS_RSA_WITH_AES_128_GCM_SHA256,TLS_RSA_WITH_AES_256_GCM_SHA384,TLS_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_256_CBC_SHA`
```

Comment:
```suggestion
If the protocol is set to HTTPS and there are no ciphers configured, the system will apply the following ciphers to support backward compatibility for TLS 1.2: `TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,TLS_RSA_WITH_AES_128_GCM_SHA256,TLS_RSA_WITH_AES_256_GCM_SHA384,TLS_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_256_CBC_SHA`
```

Reply:
i have made it small case, as the protocol that is configured in .ini file is small case. Let me know if you want to change?

Reply:
ah, then we should use tickmarks, like `https`. I'll make that change and commit my suggestion (for expediency).